backend/src/main/java/backend/mulkkam/intake/service/IntakeAmountService.java

```diff
@@ -31,6 +39,17 @@ public void modifyTarget(
         member.updateTargetAmount(intakeTargetAmountModifyRequest.toAmount());
     }
 
+    @Transactional
+    public void modifyTargetByRecommended(
+            ModifyIntakeTargetAmountByRecommendRequest modifyIntakeTargetAmountByRecommendRequest,
+            Long memberId
+    ) {
+        Member member = getMember(memberId);
+        IntakeHistory intakeHistory = intakeHistoryRepository.findByMemberIdAndHistoryDate(memberId, LocalDate.now())
+                .orElseThrow(() -> new CommonException(NOT_FOUND_INTAKE_HISTORY));
+        intakeHistory.modifyTargetAmount(new Amount(modifyIntakeTargetAmountByRecommendRequest.amount()));
+    }
+
```

Comment:
```suggestion
    @Transactional
    public void modifyTargetByRecommended(
            ModifyIntakeTargetAmountByRecommendRequest modifyIntakeTargetAmountByRecommendRequest,
            Long memberId
    ) {
        IntakeHistory intakeHistory = intakeHistoryRepository.findByMemberIdAndHistoryDate(memberId, LocalDate.now())
                .orElseThrow(() -> new CommonException(NOT_FOUND_INTAKE_HISTORY));
        intakeHistory.modifyTargetAmount(new Amount(modifyIntakeTargetAmountByRecommendRequest.amount()));
        intakeHistoryRepository.save(intakeHistory);
    }

```

다른 PR 코멘트에도 비슷한 내용을 남긴 것 같긴 한데, `Amount` 필드가 불변 타입이라서 더티 체킹이 안 될수도 있을 것 같아요!

그리고 `Member` 조회는 토큰 도입 이후 `ArgumentResolver`에서 해주기 때문에 불필요할 것 같다는 생각이 듭니다!

Reply:
현재는 리졸버가 없어서 해당과 같이 안하면 컴파일이 오류가 나기 때문에 머지할 때 넣도록 하겠습니다~